src/_experimental.scss

```diff
@@ -159,3 +169,102 @@
 		}
 	}
 }
+
+
+.provisional {
+	.alert-info > :first-child::before,
+	.alert-success > :first-child::before,
+	.alert-warning > :first-child::before,
+	.alert-danger > :first-child::before {
+		content: none;
+	}
+
+	.alert {
```

Comment:
Along with the markup change proposed, you can use the parent selector ```&:```

fyi ref. https://sass-lang.com/documentation/style-rules/parent-selector

Reply:
Not sure how to action this one. 

Did you mean on the .alert-info > :first-child::before, .alert-success > :first-child::before, etc?

Should that be .alert-info, .alert-success, .alert-warning, .alert-danger { &> :first-child::before { ?

Reply:
The code will look like this
```
<div class="provisional alert alert-warning">
	<h2>Alerte d’avertissement</h2>
	<p>Détails optionnel d'alerte.</p>
</div>
```

And yes add it to the other CSS class name, like "alert-warning, ..."

Reply:
Addressed in #1650.